### PR TITLE
CJC-593

### PR DESCRIPTION
**Pull request by ccaprio97**
## What’s changing?

Updating API docs to accurately reflect current api 

## Why?

[_Link to specific issue (ex: Closes #123)_](https://e-jira.fns.usda.net/jira-prod/browse/NAC-593)

### CODE REVIEW

**Comment by ccaprio97** [comment]
@adabbraccio-ventera  the docs have been updated. Wanted you to approve this ticket on whether or not we need to update the widdershin templates so that the .md file also contains the other match api request examples that are in the openapi.yaml file

File: match/docs/openapi/schemas/match-query.yaml [hunk 1]
```diff
@@ -16,6 +16,8 @@ MatchRequest:
       examples:
         Single person:
           $ref: '#/MatchRequestExamples/Single'
+        Single required person:
```
Comment (bgfoshay) [inline match/docs/openapi/schemas/match-query.yaml:19]: Question/Nitpick: Interested to hear what others think? But are there other options we could consider for "Single required person"? This made me think it was referring to an individual being required as opposed to a request for a single person that contains only the required fields.

Comment (ccaprio97) [reply match/docs/openapi/schemas/match-query.yaml:19]: Single person with required fields? Dont know how long it can be

**Review by bgfoshay — APPROVED** [review]

File: match/docs/openapi/schemas/match-query.yaml (hunk 1 above)
Comment (adabbraccio-ventera) [reply match/docs/openapi/schemas/match-query.yaml:19]: I agree with Bill. Your suggestion Conner was along the lines of what I was thinking.

**Pull request merged**